app/ide-desktop/lib/dashboard/src/authentication/src/dashboard/useProjectManagerAdapter.ts

```diff
@@ -0,0 +1,93 @@
+/** @file Compatible with the desktop and the web client */
+import { useMemo } from 'react'
+import { ProjectManager } from 'enso-studio-content/src/project_manager'
+import * as auth from '../authentication/providers/auth'
+import { Project, ProjectState, useBackendService } from './service'
+
+interface ProjectManagerAdapter {
```

Comment:
file is missing sections: [style guide](https://github.com/enso-org/enso/blob/develop/docs/style-guide/rust.md#sections)